dev-util/scons/scons-3.0.5-r1.ebuild

```diff
@@ -62,6 +64,13 @@ src_prepare() {
 	# and fix manpage install location
 	sed -i -e '/cmdclass/,/},$/d' \
 		-e '/data_files/s:man/:share/man/:' "${S}"/setup.py || die
+	# addtional fix for Gentoo JDK installation paths
+	sed -i "s/\/usr\/lib\/openjdk/\/usr\/$(get_libdir)\/openjdk/" "${S}"/engine/SCons/Tool/JavaCommon.py || die "failed to modify JavaCommon.py"
```

Comment:
Firstly, you should use `:` or another char to avoid backslash madness.

Secondly, can't this be achieved with `/usr/lib/jdk` path? If not, please just add `lib*` in a patch file.

Also, as this affects the engine it *must* be reported upstream.

Reply:
I added the additional "patch" with get_libdir after news about stabilisation of 17.1 profile and openjdk uses `local dest="/usr/$(get_libdir)/${PN}-${SLOT}"`. So then I actually require to test it with `lib*`